### DISCUSSION TITLE
HammerDB v4.4 Window Resize Issues

### DISCUSSION THREAD

**Discussion by mknassy** [TPC-Council/HammerDB#358] [Q&A] [answered]
New to GitHub and HammerDB, please direct me to the correct location for support if it's not here.

I am running the latest version of HammerDB v4.4 on Oracle Exadata with Linux v7.9.  HammerDB is rendered via my x-window using Xming v6.9.0.31 on my Windows 10 Pro 21H2 that has Java(TM) SE Runtime Environment (build 1.8.0_311-b11) on it.

**ISSUE:**
When I open HummerDB and select Oracle TPROC-C Build / Schema Build / Options, I am unable to resize the window (see Fig.1).  I looked around for a config file but, could not find anything that would change the window size.  I tried changing the .../config/oracle.xml file to no avail.

 theme>
         scaling>auto< /scaling>
         scaletheme>auto< /scaletheme>
         pixelsperpoint>auto< /pixelsperpoint>
 

**Fig.1**
![image](https://user-images.githubusercontent.com/105246936/167503697-8ed9d031-93c5-4a37-acfa-1db761db73cb.png)

Thanks
Martin

**Comment by sm-shaw** [comment, marked as answer]
This is an issue with your X windows environment supporting UHD scalable graphics, there may be settings you have to override the default behaviour:

Alternatively, this is in the documentation here: https://www.hammerdb.com/docs/ch01s10.html as follows:

However, some displays or third-party X Windows servers may not be updated to support scalable graphics. In this case, the scaling value can be set to fixed and a standard 96 dpi display will be used with the fixed themes from HammerDB v3.3.
```
<theme>
<scaling>fixed</scaling>
<scaletheme>auto</scaletheme>
<pixelsperpoint>auto</pixelsperpoint>
</theme>
```
